#ifndef __SHARE_RESOURCE_HANDLER_TEST_H__
#define __SHARE_RESOURCE_HANDLER_TEST_H__
#define private public
#define protected public

#include "gtest/gtest.h"
class ShareResourceHandlerTest : public testing::Test {
public:
    void SetUp()
    {
    }

    void TearDown()
    {
    }

    static void SetUpTestCase()
    {
    }

    static void TearDownTestCase()
    {
    }
};

#endif